release/ray_release/command_runner/_prometheus_metrics.py

```diff
@@ -13,6 +13,8 @@
 
 DEFAULT_PROMETHEUS_HOST = "http://localhost:9090"
 PROMETHEUS_HOST_ENV_VAR = "RAY_PROMETHEUS_HOST"
+DEFAULT_PROMETHEUS_HEADERS = "{}"
+PROMETHEUS_HEADERS_ENV_VAR = "RAY_PROMETHEUS_HEADERS"
```

Comment:
Can you also add the comments about the format here?

Reply:
I have added the comment below and next to the `parse_prom_headers` function.

Reply:
> I am not familiar with our Prometheus setup, but the Ray head node has several metric endpoints, including 8080, Autoscaler metrics, and Dashboard metrics. Are they in separate codepaths?

Yes, they are in the different code paths. Prometheus will periodically scrape metrics from these metrics endpoints, while this PR targets the endpoints of the Prometheus server.